dags/springer/dag_process_file.py

```diff
@@ -36,7 +36,7 @@ def springer_validate_record(enriched_file):
     validate(enriched_file, schema)
 
 
-@dag(start_date=airflow.utils.dates.days_ago(0))
+@dag(schedule_interval=None, start_date=airflow.utils.dates.days_ago(0))
```

Comment:
do we need the `start_date`?

Reply:
yes, we need it, otherwise, we need to put the start_date on every task:
https://airflow.apache.org/docs/apache-airflow/1.10.4/_modules/airflow/models/dag.html

Reply:
```def add_task(self, task):
        """
        Add a task to the DAG

        :param task: the task you want to add
        :type task: task
        """
        if not self.start_date and not task.start_date:
            raise AirflowException("Task is missing the start_date parameter")
        # if the task has no start date, assign it the same as the DAG
        elif not task.start_date:
            task.start_date = self.start_date
        # otherwise, the task will start on the later of its own start date and
        # the DAG's start date
        elif self.start_date:
            task.start_date = max(task.start_date, self.start_date)```